translations/Hindi/general-rules.md

```diff
@@ -0,0 +1,39 @@
+# General Rules for Translation in Hindi
+
+## Overview
+
+* Hindi is a very flexible & simple language. Since it is very much like English, the translations are an easy task.
+
+* Translations can be either formal or informal, but formal translations are preferred for corporate work.
+
+* The translations can be done in any person (first, second or third) according to the content one wants to translate.
+  <br> **Note:** For usage in zulip where we give instructions to the user, it would be recommended to use first person in formal. This will also maintain consistency with the other sites that the user may browse.
+
+* Similarly, the desicion to use the singular or plural forms also depends upon the type of content.
+
+* The usage of verbs & nouns is similar to English, the type of the verb & the noun used changes according to the construction of the sentence & the content, so there are no such rules for verbs & nouns.
+  <br> **Note:** In languages similar to Hindi, it is very common that the root word gets a suffix or is changed based on the sentence a lot more frequently than English. This is **NOT** the case with Hindi, therefore care has to be taken while translating that the word has been converted to the correct form properly.
```

Comment:
Usage of <br> should be removed.

Reply:
If it's the br tag, they've been removed. If you meant something different, it's not visible/replaced with a new line